### PR TITLE
fix an intermittent bug. needs to be refactored out.

### PR DESCRIPTION
**Pull request by michaelmarshall**

### CODE REVIEW

File: test/project.e2e-spec.ts [hunk 1]
```diff
@@ -587,6 +587,12 @@ describe('Project e2e', () => {
         id: project.id,
       }
     );
+
+    // tech debt: this prevents an intermittant bug
+    setTimeout(() => {
+      //
+    });
```
Comment (CarsonF) [inline test/project.e2e-spec.ts:594]: Does it? Or does it do nothing and you just aren't experiencing the _intermittent_ bug?

Comment (michaelmarshall) [reply test/project.e2e-spec.ts:594]: Well it did on my machine. Not in CI. I'd toggle it many times to check.

Comment (michaelmarshall) [reply test/project.e2e-spec.ts]: I'm giving up and skipping it. this is holding up progress.

Comment (michaelmarshall) [reply test/project.e2e-spec.ts]: I'll create a ticket to fix it.

**Pull request merged**